### PR TITLE
Searchvector automation

### PR DESCRIPTION
**Pull request by oleggator**
**What is the purpose of this Pull Request?**
Resolution of issue #425.

**What was done to achieve this purpose?**
- Added beat (celery scheduler) container to docker-compose
- Scheduled `searchvector` command execution every second day at 2am
- Add parameters to settings.py

**How to test if it really works?**
Will be printed messages like:
```
tasks_1     | [2019-10-01 18:31:00,003: WARNING/ForkPoolWorker-2] Running searchvector...
tasks_1     | [2019-10-01 18:31:00,024: WARNING/ForkPoolWorker-2] Searchvector is done
```

**Who can help reviewing it?**
@giovanisleite 

**TODO**

### CODE REVIEW

File: jarbas/celery.py [hunk 1]
```diff
@@ -1,9 +1,34 @@
...
 os.environ.setdefault('DJANGO_SETTINGS_MODULE', 'jarbas.settings')
 
 app = Celery('jarbas')
 app.config_from_object('django.conf:settings', namespace='CELERY')
 app.autodiscover_tasks()
+
+
+@app.on_after_configure.connect
+def setup_periodic_tasks(sender, **kwargs):
+    from django.core import management
```
Comment (oleggator) [inline jarbas/celery.py]: `management` may be imported only after app has been initialized

**Comment by sergiomario** [comment]
@giovanisleite please, can you review this? :-)

File: jarbas/celery.py [hunk 2]
```diff
@@ -1,9 +1,34 @@
...
+
+    @app.task(ignore_result=True)
+    def searchvector():
+        print('Running searchvector...')
+        management.call_command('searchvector')
+        print('Searchvector is done')
```
Comment (cuducos) [inline jarbas/celery.py]: Can we use a propper log instead of `print`?

File: jarbas/celery.py [hunk 3]
```diff
@@ -1,9 +1,34 @@
...
+
+    if settings.SCHEDULE_SEARCHVECTOR:
```
Comment (cuducos) [inline jarbas/celery.py]: What do you think about early abort to keep a [flat success path](https://www.vinta.com.br/blog/2018/flat-success-path/), that is to say, the main code in the main indentation level?

**Comment by cuducos** [comment]
Personally I believe that this PR brings an avoidable level of complexity in terms of maintainability -- but it is definitively **not** @oleggator's fault. Basically I believe we need to adopt his design (which is better than what we have) to avoid having _two_ different ways to handle scheduled jobs in our stack.

What I mean is that, nowadays, we keep a bunch of scheduled tabs in manually set [`crontab`](https://github.com/okfn-brasil/serenata-de-amor/blob/master/contrib/crontab/crontab). As @oleggator's have a lot of advantages in terms of maintainability (basically embedding the schedule thing in the application layer, and not as a server manual configuration), I would suggest for maintainers to move all cronjobs to the design suggested in this PR ; )

File: jarbas/celery.py (hunk 2 above)
Comment (oleggator) [reply jarbas/celery.py]: Replaced `print` with `logging`

File: jarbas/celery.py (hunk 3 above)
Comment (oleggator) [reply jarbas/celery.py]: Reduced indentation level

**Comment by oleggator** [comment]
Maybe I can modify existing cron scripts to run using celery?

**Pull request merged**